harvard-lts/mps-viewer
# Add mirador-harvard-custom-plugin

**Add mirador-harvard-custom-plugin**
* * *

**JIRA Tickets**: 
* [LTSVIEWER-266: Relocate plugins from ops-viewer repo](https://jira.huit.harvard.edu/browse/LTSVIEWER-266)

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)
   * PR with the plugin work: https://github.com/harvard-lts/mirador-harvard-custom-plugin/pull/1

# What does this Pull Request do?
Adds a new plugin from Harvard customizations and removes the local code references that once drove these customizations

These changes have already been built to npmjs as [@harvard-lts/mirador-harvard-custom-plugin 1.0.0](https://www.npmjs.com/package/@harvard-lts/mirador-harvard-custom-plugin).

# How should this be tested?
Spin up viewer from this branch and confirm that the following elements are present:
* Harvard shield in the lower left corner (this is just an image, not a link)
* If v2 or v3 manifest: The link underneath "Attribution" or "Provided by" (Toggle sidebar -> Rights) reads as normal text, "Copyright & Terms of Use" rather than displaying the url itself
* If legacy manifest: The link underneath "License" (Toggle sidebar -> Rights) reads as normal text, "Copyright & Terms of Use" rather than displaying the url itself


# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? No
- integration tests? No

# Interested parties
@enriquediaz
